5.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <conio.h>
+void main ()
+{
+    int y;
+   
+    printf ("Enter the year : ");
+    scanf("%d" , y);

```

Comment:
Please change y with &y

Reply:
@Anjali-1801  and @mansi
Why is this PR not geeting merged? Its now more than a week. Any issue? 
Please let me know why its still pending.

Reply:
Sir there were few errors in the first program, those were resolved yesterday only. I was about to merge it today.